Classes/KIFUITestActor.m

```diff
@@ -387,24 +387,32 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text
 
 - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView *)fallbackView
 {
+    UIView *fallback = fallbackView;
+    if (!fallback) {
+        UIResponder *firstResponder = [[[UIApplication sharedApplication] keyWindow] firstResponder];
+
+        if ([firstResponder isKindOfClass:[UIView class]]) {
+            fallback = (UIView *)firstResponder;
+        }
+    }
+
+    id<UITextInputTraits> textInputField = nil;
```

Comment:
Yea, saw it appearing on the integration tests on iOS10 and thought I'd better fix it. I could just change the app to disable autocorrection, but I figured if I'm already in here, I might as well fix it for all text controls.